docs/vaahstore/index.md

```diff
@@ -0,0 +1,51 @@
+# VaahStore
+
+[[toc]]
+
+## Introduction
+
+VaahStore is a E-Commerce framework built on top of the latest opensource technologies such as `Laravel 10`, a `PHP` framework and `Vue 3`, a progressive Javascript framework.
+
+It is suitable for all small or large E-Commerce business demands using a simple set-up procedure. Built on top of `VaahCms` and equipped with an easy product information management.
```

Comment:
@bharatkumar-j001 `Built on top of` feels review repetitive, `VaahStore` is a module for `VaahCMS`